index.js

```diff
@@ -268,6 +273,53 @@ export function setSharedWebCredentials(
   return RNKeychainManager.setSharedWebCredentialsForServer(
     server,
     username,
-    password
+    password,
+    options
   );
 }
+
+/**
+ * Inquire if the type of local authentication policy (LAPolicy) is supported
+ * on this device with the device settings the user chose.
+ * @param {object} options LAPolicy option, iOS only
+ * @return {Promise} Resolves to `true` when supported, otherwise `false`
+ */
+export function canImplyAuthentication(options?: Options): Promise<boolean> {
+  if (!RNKeychainManager.canCheckAuthentication) {
+    return Promise.resolve(false);
+  }
+  return RNKeychainManager.canCheckAuthentication(options);
+}
+
+/** ANDROID ONLY */
+
+/**
+ * (Android only) Returns guaranteed security level supported by this library
+ * on the current device.
+ * @param {object} options An Keychain options object.
+ * @return {Promise} Resolves to `SECURITY_LEVEL` when supported, otherwise `null`.
+ */
+export function getSecurityLevel(
+  options?: Options
+): Promise<?$Values<typeof SECURITY_LEVEL>> {
+  if (!RNKeychainManager.getSecurityLevel) {
+    return Promise.resolve(null);
+  }
+  return RNKeychainManager.getSecurityLevel(options);
+}
+
+/**
+ * Get what type of hardware biometry support the device has.
+ * @param {object} options An Keychain options object.
+ * @return {Promise} Resolves to a `BIOMETRY_TYPE` when supported, otherwise `null`
+ */
+export function getSupportedBiometryType(
+  options?: Options
+): Promise<?$Values<typeof BIOMETRY_TYPE>> {
+  if (!RNKeychainManager.getSupportedBiometryType) {
+    return Promise.resolve(null);
+  }
+  return RNKeychainManager.getSupportedBiometryType(options);
```

Comment:
This change will either need to update the REACT_METOD for the ios native module, as it expect zero arguments or a simple check here will need to be added. 
```javascript
  if (Platform.OS === 'android') {
    return RNKeychainManager.getSupportedBiometryType(options);
  }
  if (Platform.OS === 'ios') {
    return RNKeychainManager.getSupportedBiometryType();
  }
```

Reply:
ios version does not exist for that method.

`RNKeychainManager.getSupportedBiometryType` will be `undefined` for iOS platform

Reply:
fixed! Thanks for catching that!